<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>596</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ManeSystem_version_0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="left" native="true">
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>275</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#left{
background-color: rgb(255, 255, 255);}</string>
      </property>
      <widget class="QWidget" name="password" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>231</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#password{
	background-image: url(:/img/xun.png);
background-repeat:no-repeat

}</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordLine">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>6</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/key.png);</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="username" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>231</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#username{
	background-image: url(:/img/xun.png);
background-repeat: no-reoeat
}</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="usernameLine">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/img/favicon.ico);</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MS工资管理系统</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>221</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="midLineWidth">
        <number>4</number>
       </property>
       <property name="text">
        <string>*登陆管理员账号以获取软件使用权限
*系统会自动计算个人所得税，应发数，实发数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);

	
	
	background-color: rgb(77, 154, 231);
	
	border-radius:10px
}
QPushButton:pressed{
		background-color: rgb(46, 117, 209);
}
</string>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>400</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);

	
	
	background-color: rgb(152, 152, 152);
	
	border-radius:10px
}
QPushButton:pressed{
		
	background-color: rgb(27, 27, 27);
}
</string>
       </property>
       <property name="text">
        <string>忘记密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="loginError">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string> 登陆失败  请输入正确用户名和密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <widget class="QPushButton" name="zhedie">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>20</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/img/zhedie.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right" native="true">
      <property name="minimumSize">
       <size>
        <width>571</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111846</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#right{
background-color: rgb(0, 0, 127);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="list" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>541</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>541</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#list{
background-color: rgb(255, 255, 255);
border-radius:15px
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>29</number>
          </property>
          <property name="rightMargin">
           <number>29</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <widget class="QLineEdit" name="find">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>161</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
            <widget class="QPushButton" name="findButton">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>30</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	
	

	border-radius:10px;
	
	background-color: rgb(77, 154, 231);
	color: rgb(255, 255, 255);
	
}
QPushButton:pressed{
		background-color: rgb(46, 117, 209);
}
</string>
             </property>
             <property name="text">
              <string>查找</string>
             </property>
            </widget>
            <widget class="QPushButton" name="addButton">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>30</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-image: url(:/img/add.png);

border-radius:15px
}
QPushButton:pressed{
	
	background-color: rgb(255, 170, 255);
			
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="delete_2">
             <property name="geometry">
              <rect>
               <x>330</x>
               <y>30</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">
QPushButton{
border-image: url(:/img/delect.png);



}
QPushButton:pressed{
	
	background-color: rgb(175, 255, 252);
		
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="editButton">
             <property name="geometry">
              <rect>
               <x>380</x>
               <y>30</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">
QPushButton{
border-image: url(:/img/edit.png);

}
QPushButton:pressed{
	
	background-color: rgb(0, 0, 0);
		
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="EmpTable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>401</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="SetForm" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>401</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#SetForm
{

	background-color: rgb(0, 0, 0);
background-image: url(:/img/pen.png);
border-radius:15px
}
</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>100</number>
             </property>
             <property name="topMargin">
              <number>43</number>
             </property>
             <property name="rightMargin">
              <number>100</number>
             </property>
             <property name="bottomMargin">
              <number>32</number>
             </property>
             <item>
              <widget class="QWidget" name="middle" native="true">
               <property name="minimumSize">
                <size>
                 <width>428</width>
                 <height>237</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>428</width>
                 <height>237</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QWidget#middle{
	background-color: rgb(0, 0, 0);
}</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="labelAlignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="horizontalSpacing">
                 <number>19</number>
                </property>
                <property name="rightMargin">
                 <number>20</number>
                </property>
                <property name="bottomMargin">
                 <number>12</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="empidLabel">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>职工编号</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="empid">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="basepayLabel">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>基本工资</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="basepay">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="alloLabel">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>津贴</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="allo">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="subsidyLabel">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>补贴</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="subsidy">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="postesubLabel">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>岗贴</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="posteSub">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="punishLabel">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>应扣数</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="punish">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_7" native="true">
               <property name="minimumSize">
                <size>
                 <width>428</width>
                 <height>92</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>428</width>
                 <height>92</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="addButton22">
                  <property name="font">
                   <font>
                    <pointsize>17</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);

	
	
	background-color: rgb(0, 0, 0);
	
	border-radius:10px
}
QPushButton:pressed{
		
	
	color: rgb(255, 0, 0);
	
}
</string>
                  </property>
                  <property name="text">
                   <string>确认提交</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tipsPane" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#tipsPane{background-color: rgb(255, 255, 255);
border-radius:15px
}</string>
         </property>
         <widget class="QLabel" name="tips">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>5</y>
            <width>241</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/img/tv.png);</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
